.prettierrc

```diff
@@ -1,9 +1,11 @@
 {
-  "printWidth": 100,
-  "semi": false,
+  "printWidth": 80,
+  "semi": true,
   "singleQuote": true,
-  "bracketSpacing": false,
-  "jsxBracketSameLine": true,
-  "trailingComma": "none",
```

Comment:
Allows for better refactoring, not constantly having to add a comma when commenting something out and back in.

Reply:
And better diffs. +1!